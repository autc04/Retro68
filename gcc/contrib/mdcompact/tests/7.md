(define_insn "and<mode>3<vczle><vczbe>"
  [(set (match_operand:VDQ_I 0 "register_operand" "=w,w")
	(and:VDQ_I (match_operand:VDQ_I 1 "register_operand" "w,0")
		   (match_operand:VDQ_I 2 "aarch64_reg_or_bic_imm" "w,Db")))]
  "TARGET_SIMD"
  "@
   and\t%0.<Vbtype>, %1.<Vbtype>, %2.<Vbtype>
   * return aarch64_output_simd_mov_immediate (operands[2], <bitsize>,\
					       AARCH64_CHECK_BIC);"
  [(set_attr "type" "neon_logic<q>")]
)
