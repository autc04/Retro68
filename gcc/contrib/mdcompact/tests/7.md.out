(define_insn "and<mode>3<vczle><vczbe>"
  [(set (match_operand:VDQ_I 0 "register_operand")
	(and:VDQ_I (match_operand:VDQ_I 1 "register_operand")
		   (match_operand:VDQ_I 2 "aarch64_reg_or_bic_imm")))]
  "TARGET_SIMD"
  {@ [ cons: =0 , 1 , 2   ]
     [ w        , w , w   ] and\t%0.<Vbtype>, %1.<Vbtype>, %2.<Vbtype>
     [ w        , 0 , Db  ] << aarch64_output_simd_mov_immediate (operands[2], <bitsize>, AARCH64_CHECK_BIC);
  }
  [(set_attr "type" "neon_logic<q>")]
)
